index.js

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable */
+
+import menu from './modules/menu.js';
+import Book from './modules/Book-storage.js';
+import { DateTime } from './modules/luxon.js';
+import storebooks from './modules/storebooks.js';
+
+menu();
+class UI {
+  static displayBooks() {
+    const books = storebooks.getBooks();
+
+    books.forEach((book) => UI.addBookToList(book));
+  }
+
+  static addBookToList(book) {
+    const list = document.querySelector('#data');
+
+    const div = document.createElement('div');
+    div.className = 'data-cont';
+
+    div.innerHTML = `
+          <p>"${book.title}"</p>
+          <p>by</p>
+          <p>${book.author}</p>
+          <div><button id="remove-btn" class='delete'>Remove</button></div>
+       `;
+
+    list.appendChild(div);
+  }
+
+  static deleteBook(el) {
+    if (el.classList.contains('delete')) {
+      el.parentElement.parentElement.remove();
+    }
+  }
+
+  static clearField() {
+    document.querySelector('#title').value = '';
+    document.querySelector('#author').value = '';
+  }
+}
+
+document.addEventListener('DOMContentLoaded', UI.displayBooks);
+
+// form submission to the add button
+document.querySelector('#form').addEventListener('submit', (e) => {
+  e.preventDefault();
+  const title = document.querySelector('#title').value;
+  const author = document.querySelector('#author').value;
+
+  const book = new Book(title, author);
+
+  // Add data to screen
+  UI.addBookToList(book);
+
+  // Add book to storebooks
+  storebooks.addBook(book);
+
+  // clear all fields
+  UI.clearField();
+});
+
+// Removing book data
+const data = document.querySelector('#data');
+data.addEventListener('click', (e) => {
+  // Remove book from interface
+  UI.deleteBook(e.target);
+  // Remove book from local storage
+  storebooks.removeBook(e.target.parentElement.previousElementSibling.textContent);
+});
```

Comment:
- [ ] Removing a book is not working as intended. When the Remove button is invoked to remove an entry, the data is only removed from the front end but still persists in the local storage. Please improve upon this ensuring that the entry is also removed from the local storage in a delete process.

**Removal Error:**
![image](https://user-images.githubusercontent.com/12745474/214109275-3a5f905e-0b9f-4d38-90bf-630a8a9f8d68.png)